test/fixtures/application_details.yml

```diff
@@ -1,103 +0,0 @@
-one:
```

Comment:
@hab278 These fixtures were used to load the production defaults. They should be turned into seeds.